docs/conduit/WritingTxnsToFile.md

```diff
@@ -0,0 +1,191 @@
+command. This will create a configuration directory if we don't provide one to it, and write a skeleton config file
+there which we will use as the starting point for our pipeline. Here is the config file we currently have:
+```yaml
+# Generated conduit configuration file.
+
+# The importer is typically an algod archival instance.
+importer:
+    name: algod
+    config:
+      netaddr: "your algod address here"
+      token: "your algod token here"
+
+# One or more processors may be defined to manipulate what data
+# reaches the exporter.
+processors:
+
+# An exporter is defined to do something with the data.
+# Here the filewriter is defined which writes the raw block
+# data to files.
+exporter:
+    name: file_writer
+    config:
+      block-dir: "path where data should be written"
+```
+## Setting up our Importer
+We can see the specific set of plugins defined for our pipeline--an `algod` importer, and `filewriter` exporter.
+Now we will fill in the proper fields for these. I've got a local instance of algod running at `127.0.0.1:8080`,
+with an API token of `e36c01fc77e490f23e61899c0c22c6390d0fff1443af2c95d056dc5ce4e61302`. If you need help setting up
+algod, you can take a look at the [go-algorand docs](https://github.com/algorand/go-algorand#getting-started) or our
+[developer portal](https://developer.algorand.org/).
+
+Here is the completed importer config:
+```yaml
+importer:
+    name: algod
+    config:
+      netaddr: "http://127.0.0.1:8080"
+      token: "e36c01fc77e490f23e61899c0c22c6390d0fff1443af2c95d056dc5ce4e61302"
+```
+
+## Setting up our Processor
+
+The processor section in our generated config is empty, so we'll need to fill in that section with the proper data
+for the filter processor. We can paste in the output of our list command for that.
+```yaml
+name: filter_processor
+config:
+  # Filters is a list of boolean expressions that can search the payset transactions.
+  # See README for more info.
+  filters:
+    - any:
+      - tag: txn.rcv
+        expression-type: exact
+        expression: "ADDRESS"
+```
+Since conduit allows us to have multiple processors, we need to ensure that this is part of a list under the processors
+key. The filter processor uses the tag of a property and allows us to specify an exact value to match or a regex.
+For our use case we'll grab the address of a wallet I've created on testnet, `NVCAFYNKJL2NGAIZHWLIKI6HGMTLYXL7BXPBO7NXX4A7GMMWKNFKFKDKP4`.
+
+That should give us exactly what we want, a filter that only allows transaction through for which the receiver is my
+account. However, there is a lot more you can do with the filter processor. To learn more about the possible uses, take
+a look at the individual plugin documentation [here](./plugins/filter_processor.md).
+
+## Setting up our Exporter
+
+For the exporter the setup is simple. I'll set the directory output of my blocks to a temporary directory,
+`block-dir: "/tmp/conduit-blocks/"`.
```

Comment:
```suggestion
For the exporter the setup is simple. No configuration is necessary because it defaults to a directory inside the conduit data directory.
```